internal/core/src/query/SearchOnSealed.cpp

```diff
@@ -84,9 +84,14 @@ SearchOnSealed(const Schema& schema,
                                           field.get_dim(),
                                           query_data};
 
+    auto data_type = field.get_data_type();
     CheckBruteForceSearchParam(field, search_info);
-    auto sub_qr = BruteForceSearch(
-        dataset, vec_data, row_count, search_info.search_params_, bitset);
+    auto sub_qr = BruteForceSearch(dataset,
```

Comment:
what about modify BruteForceSearch into a template function?

Reply:
Ok, I'll try to modify it.